CIScripts/Test/TestRunner.ps1

```diff
@@ -6,13 +6,11 @@ function Invoke-TestScenarios {
     Param (
         [Parameter(Mandatory = $true)] [PSSessionT[]] $Sessions,
         [Parameter(Mandatory = $true)] [String] $TestenvConfFile,
-        [Parameter(Mandatory = $true)] [String] $TestConfigurationFile,
         [Parameter(Mandatory = $true)] [String] $TestReportOutputDirectory
     )
 
     $TestsBlacklist = @(
         # Put filenames of blacklisted tests here.
-        "vRouterAgentService.Tests.ps1"
```

Comment:
Are you sure that this test should be enabled in this PR?

Reply:
It's converted to `-Pending` later on. So it's disabled, but shows up on report as inconclusive. See http://logs.opencontrail.org/winci/github/WinContrail/contrail-win-ci-gh/923/testReport.html